spark/src/main/scala/geotrellis/spark/rasterize/RasterizeRDD.scala

```diff
@@ -0,0 +1,97 @@
+package geotrellis.spark.rasterize
+
+import geotrellis.vector._
+import geotrellis.raster._
+import geotrellis.spark._
+import geotrellis.spark.tiling._
+import geotrellis.raster.rasterize._
+import geotrellis.raster.rasterize.Rasterizer.Options
+import org.apache.spark.{HashPartitioner, Partitioner}
+import org.apache.spark.rdd._
+import scala.collection.immutable.VectorBuilder
+
+object RasterizeRDD {
+  /** Rasterize geometries using a constant value */
+  def fromGeometry[G <: Geometry, T: Numeric](
+    geoms: RDD[G],
+    layout: LayoutDefinition,
+    ct: CellType,
+    value: T,
+    options: Options
+  ): RDD[(SpatialKey, Tile)] with Metadata[LayoutDefinition] = {
+    fromGeometry(geoms, layout, ct, value, new HashPartitioner(geoms.sparkContext.defaultParallelism), options)
+  }
+
+  /** Rasterize geometries using a constant value */
+  def fromGeometry[G <: Geometry, T: Numeric](
+    geoms: RDD[G],
+    layout: LayoutDefinition,
+    ct: CellType,
+    value: T,
+    options: Options,
+    numPartitions: Int
+  ): RDD[(SpatialKey, Tile)] with Metadata[LayoutDefinition] = {
+    fromGeometry(geoms, layout, ct, value, new HashPartitioner(numPartitions), options)
+  }
+
+  /** Rasterize geometries using a constant value */
+  def fromGeometry[G <: Geometry, T: Numeric](
+    geoms: RDD[G],
+    layout: LayoutDefinition,
+    ct: CellType,
+    value: T,
+    partitioner: Partitioner,
+    options: Options
+  ): RDD[(SpatialKey, Tile)] with Metadata[LayoutDefinition] = {
+    val intValue = implicitly[Numeric[T]].toInt(value)
+    val dblValue = implicitly[Numeric[T]].toDouble(value)
+    val options = Options(includePartial=true, sampleType=PixelIsArea)
```

Comment:
Is there a certain reason why you defined `options` in this method when it's already being passed in as a parameter (line 44)?